README.md

```diff
@@ -29,6 +29,12 @@ common Linux PKCS#11 implementations:
 
 This is NOT an official Google product.
 
+Additional make options
```

Comment:
nit: could we put backquotes around any identifiers that would appear in code (e.g `PKCS11_LONGTYPE=32`, `CK_LONG`, `uint32_t`)